src/config/dynamic-theme-fixes.config

```diff
@@ -11802,13 +11802,15 @@ body,
 nymag.com
 
 INVERT
-.logo-svg
+.article-feed-header svg
 .article-nav-top-center
 #latest-feed-title
+.logo-svg
 #site-feed-curbed-title > .feed-title-wrapper
 .tab-trigger > svg
 .tab-trigger > svg > circle
 .tab-trigger > svg > g > circle
+.title-logo
 #vulture-trigger > svg > g > path:nth-child(1)
 #wwwthecut-trigger > svg > .active > path
```

Comment:
Could you add:

```
CSS
.collection-articles-lede_first-featured .lede-item[data-track-index="0"] .lede-image-wrap {
    z-index: 0 !important;
}
.collection-articles-lede_first-featured .lede-item[data-track-index="0"] .lede-text-wrap {
    z-index: 1 !important;
    position: absolute !important;
}

Reply:
Done. Thanks!

Reply:
Question for you:

Whenever I want to add an additional change to a pull request via the GitHub web UI, the process seems a bit work-intensive.  Currently, I:

1. Open my fork
2. Find the correct patch
3. Find the correct file (for large projects, navigating through the file structure can require quite a few clicks)
4. Edit the file
5. Save the changes

It seems like there must be a workflow using their web UI that doesn't require all those steps.

Maybe you know of a more efficient way to do it?

Reply:
> Open my fork
Find the correct patch
Find the correct file (for large projects, navigating through the file structure can require quite a few clicks)

If I type "dynamic" in the search bar, the first suggested URL is `https://github.com/darkreader/darkreader/edit/master/src/config/dynamic-theme-fixes.config` and I have a few more of such files that are into my muscle memory in combination of suggested URL's from Firefox. 

Then it's simply searching where the fix should be placed, write a small PR message and poof!